classes/statistics/PKPStatisticsHelper.inc.php

```diff
@@ -58,6 +58,9 @@
 define('STATISTICS_YESTERDAY', 'yesterday');
 define('STATISTICS_CURRENT_MONTH', 'currentMonth');
 
+// Constant used for submission files that are not full texts
+define('ASSOC_TYPE_SUBMISSION_FILE_COUNTER_OTHER', 0x0000213);
```

Comment:
I suggest moving this definition near the other `ASSOC_TYPE_`... defines. That'll make it less likely that someone will come along and think it's an unused number.